dojo/unittests/tools/test_pmd_parser.py

```diff
@@ -0,0 +1,24 @@
+from django.test import TestCase
+from dojo.tools.pmd.parser import PmdParser
+from dojo.models import Test
+
+
+class TestPMDParser(TestCase):
+
+    def test_parse_file_with_no_vuln_has_no_findings(self):
+        testfile = open("dojo/unittests/scans/pmd/pmd_no_vuln.csv")
+        parser = PmdParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(0, len(findings))
+
+    def test_parse_file_with_one_vuln_has_one_findings(self):
+        testfile = open("dojo/unittests/scans/pmd/pmd_one_vuln.csv")
+        parser = PmdParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(1, len(findings))
+
+    def test_parse_file_with_multiple_vuln_has_multiple_finding(self):
+        testfile = open("dojo/unittests/scans/pmd/pmd_many_vulns.csv")
+        parser = PmdParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(9, len(findings))
```

Comment:
Please add more check on the attributes of some findings.
Like for ex test on file_path / line attributes